### ISSUE TITLE
https://dgraphcloud.io - DGraph as a Service (Managed DB)

### ISSUE THREAD

**Issue by markmssd** [dgraph-io/dgraph#3896]
Hi all,

**TLDR:** Would you be interested in a “**DGraph as a Service**”? https://dgraphcloud.io :zap:

We’ve struggled a lot back then for installing DGraph, until we finally got it and use it in production.

We wished there was a Managed DGraph service that we could have used right away.

Since there is a LOT of issues about installing/running DGraph, we have decided to use all that we’ve learned bringing DGraph to production and create such a service.

This post is to see how interested users would be in such a service.

Here is the website: https://dgraphcloud.io

We would appreciate any comment/critique/insult/feature request 😄 

Cheers :clinking_glasses:

**Comment by campoy** [comment]
Hi there, @markmssd 

I'd love to know more about the kind of issues you've found while running Dgraph.
I know that we're already scheduling a chat with you sometime soon, so this issue can be closed.
